{"v":0,"level":30,"name":"newrelic","hostname":"santiagos-mbp","pid":85041,"time":"2020-11-01T20:40:47.374Z","msg":"Using New Relic for Node.js. Agent version: 6.14.0; Node version: v14.14.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"santiagos-mbp","pid":85041,"time":"2020-11-01T20:40:47.539Z","msg":"Agent state changed from stopped to starting."}
{"v":0,"level":30,"name":"newrelic","hostname":"santiagos-mbp","pid":85041,"time":"2020-11-01T20:40:48.465Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"santiagos-mbp","pid":85041,"time":"2020-11-01T20:40:48.465Z","msg":"Agent state changed from starting to connecting."}
{"v":0,"level":50,"name":"newrelic","hostname":"santiagos-mbp","pid":85041,"time":"2020-11-01T20:40:49.085Z","msg":"The New Relic collector rejected this agent.","component":"collector_api"}
{"v":0,"level":30,"name":"newrelic","hostname":"santiagos-mbp","pid":85041,"time":"2020-11-01T20:40:49.085Z","msg":"Agent state changed from connecting to errored."}
{"v":0,"level":50,"name":"newrelic","hostname":"santiagos-mbp","pid":85041,"time":"2020-11-01T20:40:49.086Z","msg":"New Relic for Node.js halted startup due to an error:","stack":"Error: Failed to connect to collector\n    at onStartConnect (/Users/santiagoestrago/Documents/ORT/node-chassis-asp-example/node_modules/newrelic/lib/agent.js:236:18)\n    at retry (/Users/santiagoestrago/Documents/ORT/node-chassis-asp-example/node_modules/newrelic/lib/collector/api.js:115:14)\n    at onPreConnect (/Users/santiagoestrago/Documents/ORT/node-chassis-asp-example/node_modules/newrelic/lib/collector/api.js:167:14)\n    at Immediate._onImmediate (/Users/santiagoestrago/Documents/ORT/node-chassis-asp-example/node_modules/newrelic/lib/collector/parse-response.js:67:24)\n    at processImmediate (internal/timers.js:461:21)\n    at process.callbackTrampoline (internal/async_hooks.js:129:14)","message":"Failed to connect to collector"}
